reference/5.1/Microsoft.PowerShell.Core/About/about_Types.ps1xml.md

```diff
@@ -71,34 +73,32 @@ types.
 
 ## The TypeData Cmdlets
 
-The following TypeData cmdlets are included in the Microsoft.PowerShell.Utility
-module in Windows PowerShell 3.0 and later versions of Windows PowerShell
-and PowerShell Core.
+The following TypeData cmdlets are included in the
```

Comment:
Should TypeData be shown as `*-TypeData`?

Reply:
Fixed